symantec-messaging-gateway/info.json

```diff
@@ -1,349 +1,347 @@
 {
-    "name": "symantec-messaging-gateway",
-    "version": "1.1.0",
-    "label": "Symantec Messaging Gateway",
-    "description": "Symantec Messaging Gateway is an email security solution which provides inbound and outbound messaging security. Also it can perform containment and corrective actions like block Domain/Email/IP or unblock Domain/Email/IP",
-    "publisher": "",
-    "icon_small_name": "small.png",
-    "icon_large_name": "large.png",
-    "category": "Email Gateway",
-    "configuration": {
-        "fields": [
-            {
-                "title": "Server URL",
-                "required": true,
-                "editable": true,
-                "visible": true,
-                "type": "text",
-                "name": "base_url"
-            },
-            {
-                "title": "Username",
-                "required": true,
-                "editable": true,
-                "visible": true,
-                "type": "text",
-                "name": "user_name"
-            },
-            {
-                "title": "Password",
-                "required": true,
-                "editable": true,
-                "visible": true,
-                "type": "password",
-                "name": "password"
-            },
-            {
-                "title": "Verify SSL",
-                "required": false,
-                "editable": true,
-                "visible": true,
-                "type": "checkbox",
-                "name": "verify_ssl",
-                "value": true
-            }
-        ]
+  "name": "symantec-messaging-gateway",
+  "label": "Symantec Messaging Gateway",
+  "version": "1.1.0",
+  "description": "Symantec Messaging Gateway is an email security solution which provides inbound and outbound messaging security. Also it can perform containment and corrective actions like block Domain/Email/IP or unblock Domain/Email/IP",
+  "publisher": "Fortinet",
+  "cs_approved": true,
```

Comment:
Is this a community submission?

Reply:
FortiSOAR QA is going to certify the connector.